woocommerce/woocommerce-ios
# Store creation: Update wording for existing profiler questions and summary

Closes: #10409 


## Description

This PR updates the existing profiler questions' wording to reflect the latest design. I think it's not worth the complication to update the wording only with the feature flag `optimizeProfilerQuestions` turned on - so the update is applied to the existing version of the profiler flow as well.

Changes include:
- Updated content for the free trial summary screen and subscriptions view (as subscriptions use the same model as the free trial summary screen).
- Updated header text for the selling status, category, and country questions to display a static text "About your store" instead.
- Updated title and subtitle for selling status, category, and country questions.
- Set the default style for the profiler answers to `body` style instead of `headline`.

## Testing instructions

- Enable the feature flag `optimizeProfilerQuestions` and build the app.
- Log in to your account and switch to Menu tab.
- Open the store picker and tap Add a store > Create a new store.
- Notice that the free trial summary screen displays the updated content.
- Tap Try For Free, after a new store is created, the profiler flow should be displayed. Notice that the wording is updated for each question.
- After store creation completes, the app should navigate to the My Store screen.
- Switch to the Menu tab and select Subscriptions. The content should be the same as in the updated free trial summary screen.

## Screenshots

| Free trial summary | Subscriptions | 
| ----- | ----- |
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/14cf6171-6fe6-4c78-936d-c3526b4bc1e6" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/355aa372-e34f-498b-87df-4f6626fefbfe" width=320 /> | 

Profile questions:


https://github.com/woocommerce/woocommerce-ios/assets/5533851/56a873a8-0077-410e-a30d-27e323251a8f




---
